andela/pippin-ah-frontend
# [Feature: #163966307] New Password Component

#### What does this PR do?
- This PR completes the feature for `Password Reset`

#### Description of Task to be completed?
- When a user clicks on the link sent to his/her email, they should be redirected to a `new password` page where they can reset their password.

#### How should this be manually tested?
- deploy this branch `Feature/163966307/new-password-component`
- navigate to `/resetpassword`
- enter a valid email address
- you should recieve a mail with a link
- click the link, and you should be redirected to a `New Password` page
- fill the form with matching `passwords`
- your password reset should be successful

#### What are the relevant pivotal tracker stories?
- [#163966307](https://www.pivotaltracker.com/story/show/163966307)
